vueui/src/components/view/security/guntracker.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <h3>Firearm Control System</h3>
+    <template v-if="screen">
+      <vui-button :params="{ lock: 1 }">Unlock Console</vui-button>
+    </template>
+    <template v-else>
+      <vui-button :params="{ lock: 1 }">Lock Console</vui-button>
+      <h3>Detected Firearms</h3>
+      <table>
+        <tr>
+          <th>User</th>
+          <th>Firearm</th>
+          <th/>
+        </tr>
+        <tr v-for="gun in wireless_firing_pins" :key="gun.ref">
+          <td>{{ gun.registered_info }}</td>
+          <td>{{ gun.gun_name }}</td>
+          <td>
+            <template v-if="gun.disabled_state">
+              <vui-button class="button selectedoption" :params="{ togglepin2: gun.ref }">Disabled</vui-button>
+            </template>
```

Comment:
```suggestion
          <vui-button :class="{'button' : 1, 'selectedoption' : gun.disabled_state}" :params="{ togglepin2: gun.ref }">Disabled</vui-button>
```

I think something like that should work for the class. If not check out https://vuejs.org/v2/guide/class-and-style.html

You should be able to condense all these down like this.

Reply:
Works perfectly after some testing. Thanks!